packages/devtools_shared/test/server/server_api_test.dart

```diff
@@ -12,7 +12,7 @@ import 'package:shelf/shelf.dart';
 import 'package:test/test.dart';
 
 void main() {
-  test('handle deeplink api androidBuildVariants', () async {
+  test('handle deeplink api ${DeeplinkApi.androidBuildVariants}', () async {
```

Comment:
nit: I think now the "deeplink api" text is redundant since `'${DeeplinkApi.androidBuildVariants}'` should output "DeeplinkApi.androidBuildVariants".

Reply:
The '${DeeplinkApi.androidBuildVariants}' should be evaluate to something like  `'api/androidBuildVariants'`? so I think the deeplink api is still relevant